docs/components/cloud-console/manage-organization/manage-users.md

```diff
@@ -12,24 +12,24 @@ If several users need access to the same Zeebe cluster, all users can be assigne
 
 ## Users
 
-Under this setting, members of the current organization can be managed. A user can have one of the following roles:
+The first user in an organization is the owner of the organization. An owner has all rights in an organization and can manage all settings accordingly. An organization cannot have more than one owner.
 
-- **Owner**: Owner of the organization (currently limited to one user and cannot be changed by the user.)
-- **Admin**: Restricted rights for user management.
-- **Member**: Can manage Zeebe Clusters, Client, and use Operate.
+To change the owner of the organization, utilize the user administration. The current owner selects another member of the organization, and selects **Assign as Owner** from the menu. In the dialog that appears, select which new roles are to be assigned to the current owner.
 
-The following table illustrates the rights of each role:
+### Roles and permissions
 
-|                              | Owner | Admin | Member |
-| ---------------------------- | ----- | ----- | ------ |
-| Manage Zeebe clusters        | X     | X     | X      |
-| Manage clients               | X     | X     | X      |
-| Use Operate                  | X     | X     | X      |
-| Users: Manage members        | X     | X     |        |
-| Billing: Manage reservations | X     | X     |        |
-| Billing: Request paid plan   | X     | X     |        |
-| Users: Manage admins         | X     |       |        |
+In addition to the owner, the **Admin** role is available as a second role with comprehensive rights. The admin role has the same rights as the owner, with the difference that an admin cannot manage other admins.
+
+The following roles are additionally available, providing dedicated rights for specific elements in Camunda Cloud.
+
+- **Operations Engineer**: Full access to Console and Operate, except cluster deletion privileges
+- **Analyst**: Full access to Optimize and read-only access to clusters
+- **Task User**: Full access to Tasklist and read-only access to clusters
+- **Developer**: Full access to Console, except deletion privileges; full access to Operate and Tasklist
+- **Visitor**: Read-only access
+
```

Comment:
@urbanisierung could we explain Web Modeler here too?

Reply:
@felix-mueller is `Web Modeler` a new role? Currently the backend only supports the following _new_ roles:

```
export enum OrganizationRole {
  MEMBER = 'member', // legacy
  ADMIN = 'admin',
  OWNER = 'owner',
  SUPPORTAGENT = 'supportagent',
  OPERATIONS_ENGINEER = 'operationsengineer',
  TASK_USER = 'taskuser',
  ANALYST = 'analyst',
  DEVELOPER = 'developer',
  VISITOR = 'visitor',
}
```

Reply:
according to https://docs.google.com/document/d/1FZYFn9fK2sn6ksjh7drSNad1vyMEIwK6p4KlG353yfM/edit#heading=h.aw86sy4n19y7 most roles are able to access Modeler. I was just wondering if we want to clarify this in the docs here 
@CatalinaMoisuc any thoughts? do you cover this elsewhere in the docs?